docs-mslearn/framework/understand/ingestion.md

```diff
@@ -108,6 +108,21 @@ At this point, you have a data pipeline and are ingesting data into a central da
 
 <br>
 
+## Data Ingestion Metrics
+
+| **Category** | **Definition** | **KPI** |
```

Comment:
Are these categories actual categories or are they names for the KPIs? They're all different, which makes them not very useful as a categorization tool.

Reply:
Moreso names for the KPIs, "Category" was the first to come to mind when naming the column. Maybe this can be "KPI", and the third column can be "Measurement" or something along those lines.

Reply:
I like that better. But maybe "Formula" for the third column?